sample-cap/srv/ai-api/ai-api-service.cds

```diff
@@ -0,0 +1,5 @@
+@path: 'ai-api'
+service AiApiService {
+  // Technically, it should return an array of AiDeployment entity if we define it in the model.
```

Comment:
[q] So why is it `String` then?

Reply:
😆 Okay, stringified. Now, it is really a string to keep it also technically correct.